### ISSUE TITLE
Have not integrated sqoop yet?

### ISSUE THREAD

**Issue by youfans** [EsharEditor/ambari-hue-service#18]
Have not integrated sqoop yet?

**Comment by EsharEditor** [comment]
Yes, the version of sqoop in Ambari is different from  the version in Hue.(Ambari-sqoop:1.x, HUE-sqoop:2.x). So I have not integrated sqoop.

**Comment by schwxd** [comment]
May I wonder if I have sqoop2.x installed manually in HDP2.5, how to integrate sqoop in ambari-hue? I removed the 'sqoop' from app_blacklists in "package/scripts/params.py", but it doesn't work. Thanks.

**Comment by EsharEditor** [comment]
Hi schwxd:
I suggest you modify the configuration for hue in Ambari.
【hue】-【Configs】-【Advanced】-【Advanced pseudo-distributed.ini】-【[sqoop]】

**Comment by OneCricketeer** [comment]
@schwxd 

You'll need to have something like this in your config

```conf
[sqoop]
  server_url=http://SQOOP_HOST:12000/sqoop
  ## sqoop_conf_dir=/etc/sqoop2/conf
```

And can first check that `http://SQOOP_HOST:12000/sqoop/version` is accessible to be sure the server is running. 

I would also recommend you see the [Sqoop 5 Minute Demo](http://sqoop.apache.org/docs/1.99.5/Sqoop5MinutesDemo.html) as well as issue [HUE-5128](https://issues.cloudera.org/browse/HUE-5128)
